openverse_catalog/docs/adding_a_new_provider.md

```diff
@@ -0,0 +1,90 @@
+# Openverse Providers
+
+## Overview
+
+The Openverse Catalog collects data from the APIs of sites that share openly-licensed media,and saves them in our Catalog database. This process is automated by Airflow DAGs generated for each provider. A simple provider DAG looks like this:
```

Comment:
Small spacing fix, and also it might be good to reference Airflow documentation here:
```suggestion
The Openverse Catalog collects data from the APIs of sites that share openly-licensed media, and saves them in our Catalog database. This process is automated by [Airflow DAGs](https://airflow.apache.org/docs/apache-airflow/stable/concepts/dags.html) generated for each provider. A simple provider DAG looks like this:
```